code/web/src/ui/surveyModal/SurveyModal.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+//UI Component
+import { grey, grey2, white } from '../common/colors';
+import { H3 } from "../../ui/typography";
+import Card from "../../ui/card/Card";
+import { Grid, GridCell } from '../../ui/grid';
+import Button  from '../../ui/button/Button';
+
+//App Imports
+import { APP_URL } from "../../setup/config/env";
+
+//Component
+const SurveyModal = (props) => {
+    return (
+        <div style={{
+            position: 'fixed',
+            height: '100%',
+            width: '100%',
+            bottom: 0,
+            backgroundColor: grey,
+            zIndex: 2
+        }}>
+            <div style={{
+                margin: '5% auto',
+                height: '80%',
+                width: '60%',
+                maxWidth: '40rem',
+                backgroundColor: white,
+            }}>
```

Comment:
I'm not sure how much if any of this needs to be dynamically generated but it's something to think about down the road.